.github/workflows/ci.yml

```diff
@@ -114,3 +118,20 @@ jobs:
 
       - name: Run license check
         run: ./scripts/license.sh .
+
+  conventional-commits:
+    runs-on: ubuntu-latest
+    steps:
+      - name: validate
+        uses: actions/github-script@v6
+        with:
+          script: |
+            // See https://gist.github.com/marcojahn/482410b728c31b221b70ea6d2c433f0c#file-conventional-commit-regex-md
+            const regex = /^(build|chore|ci|docs|feat|fix|perf|refactor|revert|style|test){1}(\([\w\-\.]+\))?(!)?: ([\w ])+([\s\S]*)/g;
+
+            const pr = context.payload.pull_request;
+            const title = pr.title;
+
+            if (title.match(regex) == null) {
```

Comment:
Previously, if the PR has only one commit, that one commit message needs to be semantic even if the title already is. Do we still need to check that?

Reply:
nah I thought that was super annoying before 😂 because mergify while merges squashes all commits and instead uses the PR title and body in the commit message:
https://github.com/aws/copilot-cli/blob/762e6cb39cf2571c6ef93b561b4aa3358b6ac53a/.mergify.yml#L65-L68

So we should only be checking the PR title for the best experience